components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -167,7 +168,17 @@ const ContributorProfileTab = ({
           {/* <HightlightEmptyState /> */}
 
           {isError && <>An error occured</>}
-          {isLoading && <>Loading...</>}
+          {isLoading && <>
+            {Array.from({ length: 3 }).map((_, index) => (
```

Comment:
2 skeleton is enough for this... It's pointless displaying 3 when it is not visible on first load
```suggestion
            {Array.from({ length: 2 }).map((_, index) => (
```